js/ui/calendar/ui.calendar.js

```diff
@@ -1139,24 +1213,39 @@ const Calendar = Editor.inherit({
     },
 
     _updateViewsValue: function(value) {
-        const newValue = value ? new Date(value) : null;
+        const multiselect = this.option('multiselect');
+
+        if(multiselect) {
+            this._updateViewsOption('values', value);
+        } else {
+            const newValue = value[0] ? new Date(value[0]) : null;
+            this._updateViewsOption('value', newValue);
+        }
 
-        this._view.option('value', newValue);
-        this._beforeView && this._beforeView.option('value', newValue);
-        this._afterView && this._afterView.option('value', newValue);
+    },
+
+    _updateViewsOption: function(optionName, newValue) {
+        this._view.option(optionName, newValue);
+        this._beforeView && this._beforeView.option(optionName, newValue);
+        this._afterView && this._afterView.option(optionName, newValue);
     },
 
     _updateAriaSelected: function(value, previousValue) {
-        value = value ?? this._dateOption('value');
+        const multiselect = this.option('multiselect');
 
-        const $prevSelectedCell = this._view._getCellByDate(previousValue);
-        const $selectedCell = this._view._getCellByDate(value);
+        if(!multiselect) {
+            value = value ? value : [this._dateOption('value')];
```

Comment:
Please, simplify all constructions
```suggestion
            value ??= [this._dateOption('value')];
```